wallet/res/values-ar/strings.xml

```diff
@@ -35,10 +35,8 @@
 	<string name="wallet_transactions_options_filter_all">كلاهما</string>
 	<string name="wallet_transactions_options_filter_received">تم الاستلام</string>
 	<string name="wallet_transactions_options_filter_sent">تم الارسال</string>
-	<string name="wallet_options_safety_menu">السلامة</string>
 	<string name="wallet_options_encrypt_keys_set">تعيين الرقم السري الشخصي للإنفاق</string>
 	<string name="wallet_options_encrypt_keys_change">تغيير الرقم السري الشخصي للإنفاق</string>
-	<string name="wallet_options_safety">ملاحظات السلامة</string>
```

Comment:
We don't need to delete strings in other languages.  This will be handled when we pull updates from Transifex.

Reply:
We only need to remove them in the source language (en).